Sources/SwiftProcessing/Core/Slider.swift

```diff
@@ -2,37 +2,101 @@
 import Foundation
 import UIKit
 
-open class Slider: UIKitControlElement {
+open class Slider {
```

Comment:
For future work, I think it'd make sense for this to return to inheriting from UIKitControlElement based on our conversation. 

Some of these changes could make sense in the parent class too!